TrackParamFilter/sections/general_url.txt

```diff
@@ -47,6 +47,7 @@ $removeparam=utm_social-type
 $removeparam=utm_source
 $removeparam=utm_swu
 $removeparam=utm_term
+$removeparam=utm_keyword
```

Comment:
I'm not sure if we should add it globally, nor can I find information about this parameter. @Alex-302 @AdamWr what do you think?

Reply:
According to this website - [https://larryludwig.com/utm-parameters/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https://larryludwig.com/utm-parameters/) it's optional parameter:
>Used for paid search to note what keywords were used to land on your website.

So, I guess that we can add it.